### PR TITLE
Organize steps

### PR DESCRIPTION
**Pull request by malakai97**
I'm fairly happy with this. I didn't actually set out to organize the steps, I was just looking to unify some of our language.

This involved some slight changes to check_deposit_status.feature. Some
work that was previously done behind the scenes in the steps can now be
seen as an additional When clause for some scenarios.

### CODE REVIEW

File: features/check_deposit_status.feature [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
-    When I check the status of the artifact
+    When I check on the artifact
+    And I check on the progress processing the artifact
     Then I receive a report that the artifact is in progress
 
   Scenario: Check on an invalid artifact that has been uploaded
     Given an uploaded Bentley audio artifact that failed validation
     And I am a Bentley audio content steward
-    When I check the status of the artifact
+    When I check on the artifact
```
Comment (botimer) [inline features/check_deposit_status.feature]: Same as above

File: features/check_deposit_status.feature [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
     Then I receive a report that the artifact is preserved
 
   Scenario: Check on an artifact that has been uploaded but isn't preserved yet
     Given an uploaded but not yet preserved Bentley audio artifact
     And I am a Bentley audio content steward
-    When I check the status of the artifact
+    When I check on the artifact
```
[the remark this answers is not in the document]
Comment (malakai97) [reply features/check_deposit_status.feature]: The Then clause makes assertions on both the package response and the queue_item response. Dropping the assertion on the package response--that the object is stored--does not seem like something we can do.

[the remark this answers is not in the document]
Comment (malakai97) [reply features/check_deposit_status.feature]: Originally, the single `when` clause had the behavior of both of these `when` clauses. That led to step definitions that felt more detailed than necessary.

[the remark this answers is not in the document]
Comment (botimer) [reply features/check_deposit_status.feature]: I think this is basically saying we have an API bug (two calls for a single conceptual result). We should not have to check two accounts of the same information from the outside on the basis that they might be out of sync.

I think the behavior here is that the package is stored or not, according to the package... The details of the queue are incidental.

From the client perspective, I think we would only check one thing, and that we should verify integrity at a unit level -- two examples with the same action and independent verification of the two expectations. Those would go into BagMoveJob, I think.

[the remark this answers is not in the document]
Comment (botimer) [reply features/check_deposit_status.feature]: Per discussion, we're more satisfied with looking at the deposit/queue item here.

**Pull request merged**